.github/workflows/deploy.yml

```diff
@@ -38,3 +38,18 @@ jobs:
         run: script/github-action-awscli-installation
       - name: Deploy the Serverless Demo App
         run: script/deploy-canary-demo
+  xcuitest:
```

Comment:
Understand from description we need to have deployed demo. Can we [`workflow_run`](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_run) Github action concept here?

Reply:
Can you clarify more @devalevenkatesh ?

Reply:
Yeah sure, instead triggering the new workflow here after the deploy step, wanted to check if we can separate this newly added workflow into its own file and use workflow_run to trigger the new workflow based on previous completion state. Read here: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_run.

Reply:
Sure, I think that makes more sense.

Reply:
done